pallets/fungible/src/lib.rs

```diff
@@ -57,13 +119,13 @@ pub mod pallet {
 	pub enum Error<T> {
 		/// Not Fungible item data used to mint in Fungible collection.
 		NotFungibleDataUsedToMintFungibleCollectionToken,
-		/// Not default id passed as TokenId argument
+		/// Not default id passed as TokenId argument.
```

Comment:
What is default id, 0?

Reply:
TokenId is a Newtype(but with pub field) for u32 , so yes, it's 0.

Reply:
Could you add it in?

Reply:
> Could you add it in the doc?
Done.